plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -122,6 +128,7 @@ fu_pxi_device_to_string (FuDevice *device, guint idt, GString *str)
 	fu_common_string_append_kx (str, idt, "PacketReceiptNotificationThreshold", self->prn_threshold);
 	fu_common_string_append_kv (str, idt, "SpecCheckResult",
 				    fu_pxi_device_spec_check_result_to_string (self->spec_check_result));
+	fu_common_string_append_kv (str, idt, "ModelName", g_strndup ((gchar *) self->model_name, FU_PXI_DEVICE_MODEL_NAME_LEN));
```

Comment:
you need to do something sucky like:

   g_autofree gchar *model_name = NULL;
   g_strndup ((gchar *) self->model_name, FU_PXI_DEVICE_MODEL_NAME_LEN)
   fu_common_string_append_kv (str, idt, "ModelName", model_name);

..otherwise you leak memory.